src/partials/languages/it.html

```diff
@@ -23,8 +23,13 @@
 <!-- Translations: Italian -->
 {% macro t(key) %}{{ {
   "language": "it",
+  "direction": "ltr",
```

Comment:
Please remove this, together with the search translations mentioned in https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md#missing-translations.

Reply:
ops. done. thank you